.prettierrc.yaml

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Nice catch.

Reply:
We may also want to switch to single quotes if it makes the diff smaller.

Reply:
I'm usually all in for reducing the diff, however in this case I much prefer to aim for a good code style that is well readable because whatever we do the diff will nevertheless be huge.

Reply:
If you feel like having that debate on what's "readable", please go ahead. :stuck_out_tongue:

Reply:
AFAICT the only difference between single and double quotes in JS strings is which quotes you need to escape if you want them literally in your string. Based on that:

- Apostrophes are much more common than double quotes in English, which would point in favour of double quotes for JS strings.
- JSON only allows double quotes, so using that in JS makes it easier to copy/paste between the two — a niche use-case to be sure, but still a stronger argument than "I like TYPE_OF_QUOTE better".

Reply:
I also prefer double-quotes; I would just like to move that conversation to a separate PR.

Reply:
Seems like double quotes have a good consensus behind them.

Reply:
I disagree about moving to a different PR, let's please only do the formatting once and right